### ISSUE TITLE
Breaking the API should have be done in a major version

### ISSUE THREAD

**Issue by mateusfccp** [GoogleCloudPlatform/recaptcha-enterprise-flutter#17]
Recently, 18.2.2 has been release with an API breakage.

I propose we bump the version to 19.0.0.

In our app, we had a break because of this. Although it's not something hard to change, I don't think we should break API in minor or patch versions.

**Comment by mateusfccp** [comment]
Yeah, I don't know why this has been decided this way, but then won't we have any way to know if a breaking change has been introduced?

I mean, as an API consumer I don't really care if the underlying API has changed or not, I only care for what I am directly consuming (the wrapper).

**Comment by mcorner** [comment]
Until we declare it as GA, it is subject to change (see warning at the top of the README). That said, I don't think it is likely we will change it again without a change in the underlying sdk. And when that happens we will be following more conventional semantic versioning.
